cmd/server/assets/realmadmin/show.html

```diff
@@ -23,7 +26,7 @@ <h1>Realm stats</h1>
     </p>
 
     <div class="card mb-3 shadow-sm">
-      <div class="card-header">Statistics</div>
+      <div class="card-header">Visulizations</div>
```

Comment:
```suggestion
      <div class="card-header">Visualizations</div>
```

Still not sure we really need to change the name here.